packages/cyberstorm/src/components/AntiResult/AntiResult.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import styles from "./AntiResult.module.css";
+import { classnames } from "../../utils/utils";
+
+export interface Props {
+  children?: ReactNode | ReactNode[];
```

Comment:
Why not PropsWithChildren?

Reply:
I've not liked, but I can add it there 🤷